rtloader/rtloader/api.cpp

```diff
@@ -12,7 +12,9 @@
 #ifndef _WIN32
 // clang-format off
 // handler stuff
+#ifdef __GLIBC__
```

Comment:
We support BSD backtraces as well (at least freebsd), so this is a bit restrictive I think.

Maybe a better approach would be to add a `set HAS_BACKTRACE_LIB` to the cmake section [here](https://github.com/DataDog/datadog-agent/blob/03be0e4a6456ea9a9edf6bc7b3551775c29db3e1/rtloader/rtloader/CMakeLists.txt#L31-L42) for libexec / backtrace and replace this with `#ifdef HAS_BACKTRACE_LIB`

Leaving this here for future reference/testing - could mimic llvm's structure to potentially support more BSDs: https://reviews.llvm.org/D27143#differential-review-stage
But out-of-scope for this change

Reply:
Addressed by https://github.com/DataDog/datadog-agent/pull/15256/commits/30518b6c95f98a80d7609a3e0ee9619d324d7400 and confirmed it works on Alpine at https://github.com/seqsense/datadog-agent-alpine/pull/82